File: MameInteropSDK1.0/C#/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace MameInteropTest
{
    public partial class Form1 : Form
    {
        MameInterop mame = null;

        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            mame = new MameInterop(this.textBox1);
            mame.Initialize(2, "C#Test");
        }

        private void Form1_FormClosing(object sender, FormClosingEventArgs e)
        {
            mame.Dispose();
        }
    }
}

File: MameInteropSDK1.0/VB.NET/Form1.vb
Public Class Form1
    Private Mame As MameInterop = Nothing

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        Mame = New MameInterop(Me.TextBox1)
        Mame.Initialize(2, "VBTest")
    End Sub

    Private Sub Form1_FormClosing(ByVal sender As System.Object, ByVal e As System.Windows.Forms.FormClosingEventArgs) Handles MyBase.FormClosing
        Mame.Dispose()
    End Sub
End Class
